data/en/lms_1.json

```diff
@@ -835,11 +917,11 @@
                                 "mandatory": true,
                                 "type": "Radio",
                                 "options": [{
-                                        "label": "I&apos;d like to correct their date of birth",
+                                        "label": "I’d like to correct the date of birth",
```

Comment:
Is this safe for the rest of our stack? e.g. do all browsers correctly treat our responses as utf-8?

Reply:
We're setting charset utf-8 and are using these quotes elsewhere. @armstrongb I'm guessing you aren't aware of any issues with our smart quotes not getting rendered correctly?